README.md

```diff
@@ -22,6 +22,26 @@ For people who have rusty rails, follow the following steps to setup.
 - Require the library `require "sis_importer"`
 - Import SIS Data: `SISImporter.import_sis`
 
+### Getting started
+1. Clone: `git clone https://github.com/hacsoc/the_jolly_advisor.git`
+2. New branch: `git checkout -b my-branch`
+3. Install rvm: https://rvm.io
+4. Switch to Ruby 2.2.0  
+ a. `rvm install ruby-2.2.0`  
+ b. `rvm use 2.2.0`  
```

Comment:
When you cd into the project root, rvm will detect and set the ruby version from the Gemfile, so this line isn't totally necessary, unless you have a line in your .rvmrc to disable that behavior